hera_mc/cm_dataview.py

```diff
@@ -26,41 +26,42 @@ def __init__(self, session=None):
         else:
             self.session = session
 
-    def ants_by_day(self, start, stop, interval, filename, station_types_to_check='default', output_date_format='jd'):
+    def ants_by_day(self, start, stop, time_step, output=None, station_types_to_check='default', output_date_format='jd'):
         """
         Return dated list of connected stations.
 
         Parameters:
         ------------
-        start:  start astropy.Time
-        stop:  stop astropy.Time
-        interval:  interval in days
-        filename:  filename to write to.  If stdout it will also return a dictionary
-        station_types_to_check:  e.g. HH
+        start:  start (astropy.Time)
+        stop:  stop (astropy.Time)
+        time_step:  time_step in days between start/stop (float or int)
```

Comment:
Sorry, this is still confusing to me. It looks like you want me to give you the number of days between the start & stop values (which you can just compute). Actually, I think it's the cadence at which you check which ants are connected. I do like the name `time_step` better than `interval`.